frontend/js/src/explore/artist-similarity/Data.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { useCallback, useContext, useEffect, useState } from "react";
+import tinycolor from "tinycolor2";
+import { toast } from "react-toastify";
+import SimilarArtistsGraph from "./SimilarArtistsGraph";
+import SearchBox from "./artist-search/SearchBox";
+import Panel from "./artist-panel/Panel";
+import { ToastMsg } from "../../notifications/Notifications";
+import generateTransformedArtists from "./generateTransformedArtists";
+import BrainzPlayer from "../../brainzplayer/BrainzPlayer";
+import GlobalAppContext from "../../utils/GlobalAppContext";
+
+type ArtistType = {
+  artist_mbid: string;
+  name: string;
+  comment?: string;
+  type?: string;
+  gender?: string;
+  score?: number;
+  reference_mbid?: string;
+};
+
+type MarkupResponseType = {
+  data: string;
+  type: "markup";
+};
+
+type DatasetResponseType = {
+  columns: Array<string>;
+  data: Array<ArtistType>;
+  type: "dataset";
+};
+
+type ApiResponseType = Array<MarkupResponseType | DatasetResponseType>;
+
+type NodeType = {
+  id: string;
+  artist_mbid: string;
+  artist_name: string;
+  size: number;
+  color: string;
+  score: number;
+};
+
+type LinkType = {
+  source: string;
+  target: string;
+  distance: number;
+};
+
+type GraphDataType = {
+  nodes: Array<NodeType>;
+  links: Array<LinkType>;
+};
+
+const colorGenerator = (): [tinycolor.Instance, tinycolor.Instance] => {
+  const initialColor = tinycolor(`hsv(${Math.random() * 360}, 100%, 90%)`);
+  return [initialColor, initialColor.clone().tetrad()[1]];
+};
+
+const ARTIST_MBID = "8f6bd1e4-fbe1-4f50-aa9b-94c450ec0f11";
```

Comment:
We are not going to want this hardcoded.
However the server route should send an optional seed artist MBID (where we have `props=orjson.dumps(…` in the route) and support passing one as query parameters